client/src/Components/EditFormModal.tsx

```diff
@@ -57,11 +79,45 @@ const EditFormModal = (props) => {
     setNewEmail(event.target.value);
   };
 
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    // do update stuff
+  const handleChangedPhotoURL = (event) => {
+    setNewPhotoURL(event.target.value);
+  };
+
+  const handleSubmit = async (e) => {
+    try {
+      const token = await getToken();
+      let url = `http://localhost:4200/api/users/${currentUser.uid}`;
+      let data = {
+        name: newName,
+        email: newEmail,
+      };
+      await axios.put(url, data, {
```

Comment:
@ppondo instead of using axios directly in the component can you add the methods you need to moviesService? It has logic to call the right url and attach the token. The url that gets called depends on where the app is being run (either locally when doing development or on Heroku)

Reply:
@swseverance just moved it